udine1/css/style.css

```diff
@@ -1,8 +1,341 @@
-
     src: local(gotham_narrow_book), url('fonts/gothamnarrow_book.otf') format('opentype');
+}
+
+@font-face{
+	font-family: gotham_cond_bold;
+	src: local(GothamCond-Bold), url('fonts/GothamCond-Bold.otf') format('opentype');
+}
+
+@font-face{
+	font-family: gotham_narrow_bold;
+	src: local(GothamNarrow-Bold), url('fonts/GothamNarrow-Bold.otf') format('opentype');
+}
+
+/* === General === */
+
+body, p {
+	font-family: gotham_narrow_book;
+}
+
+h2 {
+	font-size: 40px;
+	margin-bottom: 20px;
+}
+
+h3 {
+	font-size: 20px;
+	margin-bottom: 20px;
+}
+
+.margin-top-40 {
+	margin-top: 40px;
+}
+
+.no-margin {
+	margin: 0 !important;
+	padding: 0 !important;
+}
+
+.margin-negative {
+	margin-left: -18px;
+	margin-right: 5px;
+}
+
+input:focus::-webkit-input-placeholder { color:transparent; }
+input:focus:-moz-placeholder { color:transparent; } /* FF 4-18 */
+input:focus::-moz-placeholder { color:transparent; } /* FF 19+ */
+input:focus:-ms-input-placeholder { color:transparent; } /* IE 10+ */
+
+/* === Parallax === */
+
+.parallax {
+    /* The image used */
+    background-image: url("../img/parallax.jpg");
+
+    /* Set a specific height */
+    height: 477px;
+
+    /* Create the parallax scrolling effect */
+    background-attachment: fixed;
+    background-position: 50% 0 !important;
+    background-repeat: no-repeat;
+	padding: 0 80px 0 80px;
+}
+
+.help, .riservata, .seleziona {
+	font-family: gotham_narrow_book;
+	box-sizing: border-box;
+	padding: 90px 20px 70px 20px;
+	font-size: 12px;
+	text-align: center;
+	color: #fff;
+}
+
+.help p, .riservata p, .seleziona p {
+	color: rgba(255, 255, 255, 0.8);
+	letter-spacing: 1.2px;
+}
+
+.seleziona_input {
+	width: 90%;
+	text-align: justify;
+	margin-top: 16px;
+}
+
+.light-blue {
+	background:#0090d2;
+	font-family: gotham_narrow_bold;
+	border-radius:30px;
+	margin-top: 10px;
+	letter-spacing: 0.5px;
+	box-shadow: 0px 10px 30px 1px rgba(28, 38, 104, 0.3);
+}
+
+.light-blue:hover {
+	background:#fff;
+	color: #0090d2;
+	font-family: gotham_narrow_bold;
+	border-radius:30px;
+	margin-top: 10px;
+	letter-spacing: 0.5px;
+	box-shadow: 0px 10px 30px 1px rgba(28, 38, 104, 0.3);
+}
+
+/* === I nostri contatti === */
+
+.contatti {
+	font-family: gotham_narrow_book;
+	box-sizing: border-box;
+	padding: 80px 60px 70px 70px;
+	font-size: 14px;
+	color: #47545c;
+	line-height: 2.0;
+}
+
+.newsletter {
+	background-image: linear-gradient(to top right,#23419b,#018dd0,#0090d2);
+	font-family: gotham_narrow_book;
+	box-sizing: border-box;
+	color: #fff;
+	padding: 10% 10% 15% 25%;
+	font-size: 12px;
+	line-height: 2.0;
+}
+
+.newsletter p {
+	vertical-align: top;
+	display: inline;
+	color: rgba(255, 255, 255, 0.8);
+}
+
+.newsletter_checkbox {
+	margin-right: 5px;
+	margin-top: 8px;
+	display: inline;
+	vertical-align: top;
+	outline: 1px solid #fff;
+}
+
+.newsletter_input, .seleziona_input {
+	background-color:rgba(0, 0, 0, 0);
+    color:#fff;
+	border: none;
+    border-bottom: 1px solid #fff;
```

Comment:
Using height with border-bottom can sometimes make elements larger than you expect